### DISCUSSION TITLE
Problem with Github Pages

### DISCUSSION THREAD

**Discussion by joshfullmer** [slidevjs/slidev#77] [Q&A] [answered]
I am admittedly quite new to Github Pages and Vite (and Slidev), so I'm not quite sure where the issue lies.

I've published my slides (WIP) to Github Pages [here](https://joshfullmer.github.io/vue3slidev), which works great. However, if I refresh or fresh visit the page with a specific slide in the URL, I get a 404. e.g. https://joshfullmer.github.io/vue3slidev/1. 

Here's the repo for the slides: https://github.com/joshfullmer/vue3slidev.

Any help would be appreciated!

**Comment by antfu** [comment, marked as answer]
Eh, looks like it's a bit tricky, but you can try with https://github.com/rafgraph/spa-github-pages

**Reply by joshfullmer** [reply]
Turns out the solution wasn't too bad! Because everything in the public folder gets added to the root directory when built, all I needed to do was to add a custom [`404.html`](https://github.com/joshfullmer/vue3slidev/blob/main/public/404.html) that just redirects back to root (essentially starting over on refresh). 

Thanks for the help!
